persistence/sql/persister_identity.go

```diff
@@ -115,6 +116,10 @@ func createIdentityCredentials(ctx context.Context, tx *pop.Connection, i *ident
 	return nil
 }
 
+func (p *Persister) injectTraitsUrl(i *identity.Identity) {
+	i.TraitsSchemaURL = fmt.Sprintf("%s/schemas/%s", p.cf.PublicListenOn(), i.TraitsSchemaID)
```

Comment:
PublicListenOn() returns e.g. `0.0.0.0:1234` - so this will probably not work. Instead, use `SelfURLsPublic` (I think that's the name)